plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -615,6 +615,8 @@ void producer_plugin::set_program_options(
           "offset of last block producing time in microseconds. Negative number results in blocks to go out sooner, and positive number results in blocks to go out later")
          ("max-scheduled-transaction-time-per-block-ms", boost::program_options::value<int32_t>()->default_value(100),
           "Maximum wall-clock time, in milliseconds, spent retiring scheduled transactions in any block before returning to normal transaction processing.")
+         ("subjective-cpu-leeway-us", boost::program_options::value<int32_t>()->default_value( config::default_subjective_cpu_leeway_us ),
```

Comment:
You could use `bpo` instead of `boost::program_options`. Also, shouldn't this be `uint32_t` and not `int32_t`?

Reply:
`bpo` <- I just copied the one above it.

It is `int32_t` because the existing `producer_plugin` `runtime_options` is 
```
fc::optional<int32_t> subjective_cpu_leeway_us;
```